docs/using-connector.md

```diff
@@ -533,6 +534,160 @@ conf.set("spark.riakts.write.bulk-size", "500")
 conf.set("spark.riak.connections.min", "50")
 ```
 
+## Spark Streaming Example
+
+The Spark-Riak Connector can be used with Spark Streaming. To demonstrate this usage, we will work through two small Scala examples. These examples are located in the [examples folder](../examples/src/main/scala/com/basho/riak/spark/examples) of the Spark-Riak Connector repo.
+
+These examples require the use of Kafka. Please install Kafka and setup a Kafka broker prior to running this example. We will assume that there is a Kafka broker running at `127.0.0.1:9092` with a topic called `streaming`. Instructions on setting up kafka topics can be found in [this guide](https://kafka.apache.org/documentation.html#quickstart). We also assume Riak TS is installed and there is a Riak TS node running at `127.0.0.1:8087`. You will need to build the connector as well. Please follow the instruction on [building the connector](./building-and-testing-connector.md#build). After setting up, there are two examples to run: a KV bucket example and a TS table example.
```

Comment:
Please fix minor typos: capitalize k in "kafka topics can be found...", add s to "instruction" in "follow the instruction..."

Technically, users don't need to build the connector--they need to build the examples. Please update "You will need to build the connector as well. Please follow the instruction on [building the connector](./building-and-testing-connector.md#build)" with that info and point to the instructions for building the examples at https://github.com/basho/spark-riak-connector/tree/master/examples#building-and-running-examplesdemos